dashboard-frontend/src/pages/NewStudyPage/components/form/Form.tsx

```diff
@@ -0,0 +1,116 @@
+import { useEffect, useState } from 'react';
+import { FieldState } from '../../../../types/form';
+import { createNameFor } from '../../../../helpers/strings';
+import useCreateStudy from '../../../../hooks/useCreateStudy';
+import Fieldset from './Fieldset';
+import { reducer } from '../../../../helpers/arrays';
+import PrimaryButton from '../../../../components/PrimaryButton';
+import useCreateStudyConf from '../../../../hooks/useCreateStudyConf';
+import { useParams } from 'react-router-dom';
+import { StudyConfResource } from '../../../../types/study';
+
+export const Form = (props: any) => {
+  const { conf, controller, isCreateStudyForm, data } = props;
+  const fetchedConfData = data?.data.data;
+  const params = useParams<{ studySlug: string }>();
+  const title = createNameFor(conf.title);
+
+  const { createStudy, errorOnCreateStudy, isLoadingOnCreateStudy } =
+    useCreateStudy();
+
+  const {
+    createStudyConf,
+    errorOnCreateStudyConf,
+    isLoadingOnCreateStudyConf,
+  } = useCreateStudyConf();
+
+  const [fieldState, setFieldState] = useState<FieldState[]>();
+  const [globalFormData, setFormGlobalFormData] = useState<
+    StudyConfResource | any
+  >({
+    fetchedConfData,
+  });
+
+  const localFormData = !isCreateStudyForm && fetchedConfData[title];
```

Comment:
I don't like `isCreateStudyForm` as a prop. Why is there distinct logic needed for that form?

Reply:
It was purely to distinguish between which react query hook to invoke on submit, if it's a create study form, run `createStudy()`, if a conf form, run `createStudyConf()`. 

You're right though, the prop isn't needed. I've refactored so that there's no longer a prop and the form simply makes a check at the `handleSubmit` to see if current `title` is `create_study`, based on whatever the boolean is it knows which hook to run.